api/v1alpha1/cdapmaster_types.go

```diff
@@ -91,6 +91,11 @@ type CDAPMasterSpec struct {
 	// To disable this service: either omit or set the field to nil
 	// To enable this service: set it to a pointer to a AuthenticationSpec struct (can be an empty struct)
 	Authentication *AuthenticationSpec `json:"authentication,omitempty"`
+	// SystemMetricsExporter is specification for the CDAP SystemMetricsExporter service that runs as a sidecar container in various pods.
+	// This is an optional service and may not be required for CDAP to be operational.
+	// To disable this service: either omit or set the field to nil
+	// To enable this service: set it to a pointer to a SystemMetricsExporterSpec struct (can be an empty struct)
```

Comment:
nit:  Add a little more info like  - "Adding this will cause metrics collection containers to be created for the services. Use `CDAPServiceSpec.EnableSystemMetrics` field to disable this container creation for specific services.

Reply:
Added